ubidots/ubidots_out.html

```diff
@@ -1,102 +1,149 @@
+         <span data-i18n="ubidots.name"></span>
+     </label>
+       <input type="text" id="node-input-name" data-i18n="[placeholder]ubidots.name" />
+   </div>
 
-    <div class="form-row">
-        <label for="node-input-device_label">
-            <i class="icon-tag"></i>
-            <span data-i18n="ubidots.device_label.label"></span>
-        </label>
-        <input type="text" id="node-input-device_label" data-i18n="[placeholder]ubidots.device_label.placeholder" />
-    </div>
+   <div class="form-row">
+       <label for="node-input-token">
+         <i class="fa fa-lock"></i>
+         <span data-i18n="ubidots.token.label"></span>
+     </label>
+       <input type="text" id="node-input-token" data-i18n="[placeholder]ubidots.token.placeholder" />
+   </div>
+
+   <div class="form-row">
+       <label for="node-input-device_label">
+         <i class="fa fa-tag"></i>
+         <span data-i18n="ubidots.device_label.label"></span>
+     </label>
+       <input type="text" id="node-input-device_label" data-i18n="[placeholder]ubidots.device_label.placeholder" />
+   </div>
+  <div class="form-row">
+    <input type="checkbox" checked id="node-input-tls_checkbox" style="display: inline-block; width: auto; vertical-align: top;"></label>
+    <label for="node-input-tls_checkbox" style="width: calc(100% - 170px); margin-left:15px" data-i18n="ubidots.tls.label"></label>
+   </div>
+
+
+   </div>
 </script>
 
 <script type="text/x-red" data-help-name="ubidots_out">
-    <p>Receives a message and uses its values to make a request to the Ubidots API.</p>
+    <p>Receives a message and uses its values to publish a request to the Ubidots API via MQTT.</p>
 
     <p>
-    Uses the <a href="https://github.com/mqttjs/MQTT.js" target="_blank">MQTT</a> to make the connection and publish
-    the packet.
+        Uses the <a href="https://github.com/mqttjs/MQTT.js" target="_blank">MQTT</a> library to make the connection and publish the packet.
     </p>
-
     <p>
-    The message can contain the following properties, all of them except the <i>payload</i> can also be configured in
-    the node settings by double clicking it:<br />
+      Double click on the node to configure all the required fields.
+    </p>
+  <ul>
+    <li>
+      <p>
+        <b> Account Type: </b>Defaults to "Ubidots" (standard account). If you have an education account choose "Ubidots for Education".
+      </p>
+    </li>
+    <li>
+      <p>
+        <b> Name: </b> Label of node in Node-Red workspace. If empty, defaults to :"Ubidots out".
+      </p>
+    </li>
+    <li>
+      <p>
+        <b> Token: </b> (required) - Token necessary to authenticate the connection to your account. To obtain your token, login on <a href="https://www.ubidots.com" target:"_blank">ubidots.com</a>, under "My Profile" click on "API Credentials". <a href="https://help.ubidots.com/en/articles/590078-find-your-token-from-your-ubidots-account" target="_blank">Tutorial</a>
```

Comment:
Do not set this `(required)` label on this field, as every other field here (Account Type, Name and Device Label) are also required but they don't have the (required) label set explicitly.